flowauth/backend/flowauth/models.py

```diff
@@ -559,15 +559,15 @@ def init_db(force: bool = False) -> None:
         return
     current_app.logger.debug("Initialising db.")
     if force:
+        # Because we're starting fresh, we don't need to migrate and can mark the
+        # created schema as current
         current_app.logger.debug("Dropping existing db.")
         db.drop_all()
-    db.create_all()
+        db.create_all()
+        flask_migrate.stamp()
+
+    flask_migrate.upgrade()
```

Comment:
This _should_ be a noop if `force == True`, but do we want in in an `else` clause just in case something goes wrong with a migration script and we need to rebuild?

Reply:
I reckon we want to boot this stuff out of the start up sequence and separate it all out from one another really, but not right now.